src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1034,13 +1036,13 @@ def get_output_embeddings(self):
 )
 class TFBartForConditionalGeneration(TFPretrainedBartModel):
     base_model_prefix = "model"
-    _keys_to_ignore_on_load_missing = [
-        r"final_logits_bias",
-    ]
     _keys_to_ignore_on_load_unexpected = [
         r"model.encoder.embed_tokens.weight",
         r"model.decoder.embed_tokens.weight",
     ]
+    _keys_to_ignore_on_load_missing = [
+        r"final_logits_bias",
```

Comment:
The logits bias can be fine-tuned, so not sure that we want to have those in `_keys_to_ignore_on_load_missing`. Or are they missing from all bart models (and thus set to 0?)

Reply:
That's a good question, I don't know enough BART to answer to the question, so I will remove that one 👍